Makefile

```diff
@@ -38,4 +39,26 @@ build:
 
 .PHONY: container
 container:
-	$(IMAGE_BUILD_CMD) build --platform=${OS}/${ARCH} -t ${IMG} .
\ No newline at end of file
+	$(IMAGE_BUILD_CMD) build --platform=${OS}/${ARCH} -t ${IMG} .
+
+tag_and_push:
+	$(IMAGE_BUILD_CMD) tag ${IMG} ${IMAGE_REPO}:${LIVE_TAG}
+	$(IMAGE_BUILD_CMD) push ${IMG} ${IMAGE_REPO}:${LIVE_TAG}
+
+fmt:
+	go fmt ./...
+
+vet:
+	go vet ./...
+
+vendor:
+	go mod tidy && go mod vendor
+
+clean: fmt vet vendor
+
+verify:
```

Comment:
why do we need hack scripts for this?

Reply:
Just matching up to lvm-operator here

Reply:
Also there Its probably not necessary, maybe we can just remove it from both

Reply:
No we do, it's used in CI for code change validation to make sure things are formatted correctly, vendored, etc.